nservicebus/messaging/databus/azure-blob-storage_config_ABSDataBus_[5.0,).partial.md

```diff
@@ -0,0 +1,25 @@
+### Configuring the BlobServiceClient
+
+There are several ways to configure the `BlobServiceClient`. 
+
+#### Using a preconfigured BlobServiceClient
+
+A fully configured `BlobServiceClient` can be set through the settings:
+
+snippet: AzureDataBusConfigureServiceClient
+
+#### Using a custom provider
+
+A custom provider can be declared that provides a fully configured `BlobServiceClient`:
+
+snippet: CustomBlobServiceClientProvider
+
+The provider is then registered in the container:
+
+snippet: AzureDataBusInjectServiceClient
+
+#### Providing a connection string and container name
+
+snippet: AzureDataBusConnectionAndContainer
+
+NOTE: The container name is optional and will be set to the default when omitted.
```

Comment:
Should we say what the default is?

Reply:
No. It's listed among the options and their default values under [behaviour section](http://localhost:5000/nservicebus/messaging/databus/azure-blob-storage?version=absdatabus_5#behavior).